src/app/app.component.ts

```diff
@@ -59,20 +60,29 @@ export class AppComponent implements OnInit {
   }
 
   ngOnInit(): void {
-
     this.countryFormGroup = this.formBuilder.group({
-      country: [{
-        value: {
+      country: [
+        {
           name: 'Deutschland',
           alpha2Code: 'DE',
           alpha3Code: 'DEU',
-          numericCode: '276'
-        }, disabled: false
-      }]
+          numericCode: '276',
+          disabled: false,
```

Comment:
`disabled` is  not part of the country interface 

it's related to the form control! 
the first version was correct

Reply:
Oops, I copied the wrong part trying to not undo the prettier 😆